src/test/datascience/uiTests/ipywidget.ui.functional.test.ts

```diff
@@ -408,32 +408,31 @@ use(chaiAsPromised);
                 assert.include(cellOutputHtml, '>100.000</td>');
             });
         });
-        test('Render ipyvolume', async function () {
-            // See bug https://github.com/microsoft/vscode-jupyter/issues/4152
-            this.skip();
-            // const { notebookUI } = await openIPyVolumeIpynb();
-            // await assert.eventually.isFalse(notebookUI.cellHasOutput(3));
-
-            // // Confirm sliders and canvas are rendered.
-            // await retryIfFail(async () => {
-            //     await notebookUI.executeCell(1);
```

Comment:
Problem 2 - This was in the wrong place.
We execute the cell & immediate look for output, the execution is slow, hence output will not be available majority of the time, hence tests would keep running & creating widgets & then destroying it over & over by executing the cell until the test times out. 
We don't need to re-execute the cells (we can leave that for retrying the test).